cluster/config-defaults.yaml

```diff
@@ -1092,3 +1092,8 @@ sysctl_settings: ""
 
 # enables/disables the minDomains field for pod topology spread.
 min_domains_in_pod_topology_spread_enabled: "true"
+
+# enable InPlacePodVerticalScaling
+#
+# https://kubernetes.io/blog/2023/05/12/in-place-pod-resize-alpha/
+enable_in_place_pod_vertical_scaling: "true"
```

Comment:
What do you plan to use it for (and how)?
I think it might be interesting for Skipper deployment to use whole dedicated node resources.

Reply:
Not sure if the feature makes sense today, there are missing support in e.g. the Vertical pod autoscaler so it's maybe less interesting today.